code/src/Components/Form.js

```diff
@@ -0,0 +1,112 @@
+import React, { useCallback, useEffect, useState } from "react";
+import TextInput from "./TextInput";
+import Dropdown from "./Dropdown";
+import RadioButton from "./RadioButton";
+import SubmitButton from "./SubmitButton";
+import Summary from "./Summary";
+
+const projects = ["Weather app", "Chatbot", "Music releas", "Survey"];
+
+const Form = () => {
+  const [name, setName] = useState("");
+  const [evaluation, setEvaluation] = useState("");
+  const [project, setProject] = useState("");
+  const [thoughts, setThoughts] = useState("");
+  const [inValid, setInValid] = useState(false);
+  const [submitted, setSubmitted] = useState(false);
+
+  // checks if the survey is complete
+  const isSurveyComplete = useCallback(() => {
+    if (name === "") {
+      return false;
+    }
+
+    if (evaluation === "") {
+      return false;
+    }
+
+    if (project === "") {
+      return false;
+    }
+
+    if (thoughts === "") {
+      return false;
+    }
+
+    return true;
+  }, [name, evaluation, project, thoughts]);
+
+  // run the function everytime a value changes and checks if its complete
+  useEffect(() => {
+    if (isSurveyComplete()) {
+      setInValid(false);
+    } else {
+      setInValid(true);
+    }
+  }, [name, evaluation, project, thoughts, isSurveyComplete]);
+
+  // when submitted, show summery.
+  if (submitted) {
+    return (
+      <Summary
+        name={name}
+        evaluation={evaluation}
+        project={project}
+        thoughts={thoughts}
+      />
+    );
+  }
+
+  return (
+    <form
```

Comment:
You should have only one return section inside of your component. You can achieve the same effect with a structure like this:
``` jsx 
  return (
    <>
       { submitted ? 
           <Summary />
             :
           <form>...
        }
    </>
  )
```